### PR TITLE
FiBo-343: change zoneid

### PR DESCRIPTION
**Pull request by Cebox82**

### CODE REVIEW

File: android-app/app/build.gradle
Comment (Jonas-Karl) [inline android-app/app/build.gradle:28]: Is it correct to escape the string quotation marks here like this `"\"Europe/Berlin\""` instead of entering it directly like this `"Europe/Berlin"`?

File: android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/dashboard/DashboardFragment.java
Comment (Jonas-Karl) [inline android-app/app/src/main/java/de/dhbw/ka/se/fibo/ui/dashboard/DashboardFragment.java:322]: Here CodeCov mentions uncovered tests. I guess it is because the unit tests currently not working

**Review by Jonas-Karl — COMMENTED** [review]
Looks good for now. I would like to run a CI job this night to see if it works :)
After that I will approve and eventually merge if no one requests changes

**Review by Jonas-Karl — APPROVED** [review]
Good job. This works really good! 

Before merging, it would still be interesting to know if this `"\"Europe/Berlin\""` is needed as it seems to be a bit special to me

File: android-app/app/build.gradle
Comment (Cebox82) [reply android-app/app/build.gradle:28]: https://stackoverflow.com/questions/30796533/how-to-generate-buildconfigfield-with-string-type
I think that should be right

**Pull request merged**
